src/mesh/curve_interpolant.jl

```diff
@@ -0,0 +1,122 @@
+
+struct GammaCurve{RealT<:Real, NNODES}
```

Comment:
Well, if it get's a docstring, it becomes visible in the docs and thus somehow part of the official API. Do we really want this? If not, the explanation should be a comment instead of a docstring.

Reply:
You're right. A comment formatted as a docstring would be optimal ;-)